.goreleaser.yml

```diff
@@ -2,9 +2,19 @@
 # Make sure to check the documentation at http://goreleaser.com
 
 builds:
-- main: ./cli
+-
+  main: ./cli
   env:
-  - CGO_ENABLED=0
+    - CGO_ENABLED=0
+  goos:
+   - linux
+   - darwin
+   - windows
```

Comment:
The defaults for goos is only linux & darwin. I've added windows to this list so that a release file will be created in the Windows format on any future releases.